### ISSUE TITLE
Lutaml is not dealing with annotations in Express

### ISSUE THREAD

**Issue by opoudjis** [metanorma/metanorma-plugin-lutaml#98]
10303-41 schema clauses are generated by:

````
= X
:lutaml-express-index: schemas; schemas.yaml;
:bare: true
:mn-document-class: iso
:mn-output-extensions: html

[lutaml, schemas, context, leveloffset=+1]
---
{% for schema in context.schemas %}
{% if schema.id == "document_schema" %}
{% assign schemaname = schema.id | append: "$" | remove: "_schema$" | remove: "$" | replace: "_", " " | capitalize %}

[%unnumbered]
= {{schemaname }}

[source%unnumbered]
--
{{ schema.source }}
--
{% endif %}
{% endfor %}
---
````

This is a straightforward iteration, and schema.source is meant to deal with all the Express stuff it finds, and generate usable Asciidoc.

It is leaving undigested annotations and nested Lutaml in the generated Asciidoc, which make it unusable.

I attach Clause 11.

[document.presentation.xml.13.html.zip](https://github.com/metanorma/metanorma-plugin-lutaml/files/12581976/document.presentation.xml.13.html.zip)

The random `(*` and `{% assign reference_froms = schema.interfaces | where: “kind”, :REFERENCE %}` are there in the initial Semantic XML that processes that Lutaml.

**Comment by ronaldtse** [comment]
This used to work, it's not a Lutaml / Expressir issue.

**Comment by opoudjis** [comment]
Will test this on annotated-express, but I do not know the internals of Express, and debugging this would be very time consuming.

**Comment by opoudjis** [comment]
Is working fine in annotated-express, so there is something about the directory structure of iso10303 that is confusing it.

**Comment by opoudjis** [comment]
This is a different issue: *_annotated.exp files are not being generated in my test repository.

**Comment by opoudjis** [comment]
I was using [iso-10303-stepmod-wg12](https://github.com/metanorma/iso-10303-stepmod-wg12)  as my starting point; I need to use [iso-10303-stepmod](https://github.com/metanorma/iso-10303-stepmod) instead.

**Comment by opoudjis** [comment]
> `*_annotated.exp` is generated by the stepmod2mn annotation process.

Yes, and that is output to [iso-10303-stepmod](https://github.com/metanorma/iso-10303-stepmod) not [iso-10303-stepmod-wg12](https://github.com/metanorma/iso-10303-stepmod-wg12). I was confused because the instructions spoke of -wg12: that is one step too early for me to be involved.

**Comment by Intelligent2013** [comment]
> `*_annotated.exp` is generated by the stepmod2mn annotation process.

@ronaldtse stepmod2mn doesn't generate `.exp`. I think it's stepmod-utils result.

**Comment by opoudjis** [comment]
Am running on the right repository, but the issue is persisting. It turns out to be a stepmod issue:

Stepmod generates 04-schemas.adoc, the template for all the schema documents. The document it *should be generating is 

`https://github.com/metanorma/annotated-express/blob/main/data/documents/resources/fundamentals_of_product_description_and_support/sections/04-schemas.adoc`

it should not be populating this 18KB Lutaml template, just reproducing it.

Instead the 04-schemas.adoc document being generated is 807KB, which appears to have resulted from jamming all the annotated express files into it. Because populating the lutaml does not seem to invoke "source" properly, comments are being left unprocessed.

The way the document is in annotated-express is working: please output just the 18KB template as 04-schemas.adoc
